plugins/dashboard/frontend/src/app/components/ExplorerTransaction.tsx

```diff
@@ -0,0 +1,149 @@
+                                                              </div>
+                                                          </ListGroup.Item>}
+                                                      </ListGroup>
+                                                  </div>
+                                              )
+                                          })}
+                                      </div>
+                                          <IconContext.Provider value={{ color: "#00a0ff", size: "2em"}}>
+                                              <div>
+                                                  <FaChevronCircleRight />
+                                              </div>
+                                          </IconContext.Provider>
+                                      <div style={style}>
+                                            <span>Outputs</span>
+                                            <hr/>
+                                            {tx.outputs.map((output, i) => {
+                                                return (
+                                                    <div className={"mb-2"} key={i}>
+                                                        <span className={"mb-2"}>Index: <Badge variant={"primary"}>{i}</Badge></span>
+                                                        <ListGroup>
+                                                            <ListGroup.Item>ID: <a href={"#"}>{output.outputID.base58}</a></ListGroup.Item>
+                                                            <ListGroup.Item>Address: <a href={"#"}> {output.address}</a></ListGroup.Item>
+                                                            <ListGroup.Item>Type: {output.type}</ListGroup.Item>
+                                                            <ListGroup.Item>Output Index: {output.outputID.outputIndex}</ListGroup.Item>
+                                                            <ListGroup.Item>
+                                                                Balances:
+                                                                <div>
+                                                                    {Object.entries(output.balances).map((entry, i) => (<div key={i}><Badge variant="success">{entry[1]} {entry[0]}</Badge></div>))}
+                                                                </div>
+                                                            </ListGroup.Item>
+                                                        </ListGroup>
+                                                    </div>
+                                                )
+                                            })}
+                                        </div>
+                                    </div>
+                                </ListGroup.Item>
+                                <ListGroup.Item>Data payload: {tx.dataPayload}</ListGroup.Item>
```

Comment:
you can write it like this:
```
{ tx.dataPayload && <ListGroup.Item>Data payload: {tx.dataPayload}</ListGroup.Item>}
```
and then if data payload is null, no empty listgroupitem is displayed.